docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,255 @@
+3. Click **Start**.
+
+From Operate, you should now notice a process instance running. You'll notice the process has stopped at **Charge credit card**, because we'll need to configure a job worker.
+
+## Step 6: Implement a service task
+
+To implement a service task, take the following steps:
+
+### Configure Spring Boot Starter
+
+See our documentation on [adding the Spring Zeebe SDK to your project](/apis-tools/spring-zeebe-sdk/getting-started.md#add-the-spring-zeebe-sdk-to-your-project) for more details, also described below:
+
+1. Copy the following code snippet into the `pom.xml` file of your Spring project, below properties and above dependencies:
+
+```
+<repositories>
+   <repository>
+      <releases>
+         <enabled>true</enabled>
+      </releases>
+      <snapshots>
+         <enabled>false</enabled>
+      </snapshots>
+      <id>identity</id>
+      <name>Camunda Identity</name>
+      <url>https://artifacts.camunda.com/artifactory/camunda-identity/</url>
+   </repository>
+</repositories>
+```
+
+2. Add the following dependency to your `pom.xml` file:
+
+```
+<dependency>
+   <groupId>io.camunda</groupId>
+   <artifactId>spring-boot-starter-camunda-sdk</artifactId>
+   <version>8.5.0</version>
+</dependency>
+```
+
+### Configure the Zeebe client
+
+Open your `src/main/resources/application.properties` file, and paste the following snippet to connect to the Self-Managed Zeebe broker:
+
+```
+zeebe.client.broker.grpcAddress=http://127.0.0.1:26500
+zeebe.client.broker.restAddress=http://127.0.0.1:8080
+zeebe.client.security.plaintext=true
+```
+
+### Create a worker
+
+1. In `src/main/java/io/camunda/demo/process_payments/`, create a file called `ChargeCreditCardWorker.java`.
+2. In the file created above, paste the following dependencies and package `package io.camunda.demo.process_payments`:
+
+```java
+package io.camunda.demo.process_payments;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import io.camunda.zeebe.spring.client.annotation.JobWorker;
+import io.camunda.zeebe.spring.client.annotation.Variable;
+```
+
+3. Next, we can decorate our class with `@Component` and instantiate a logger:
+
+```java
+@Component
+public class ChargeCreditCardWorker {
+
+  private final static Logger LOG = LoggerFactory.getLogger(ChargeCreditCardWorker.class);
+```
+
+4. Add the following method and decorate the `chargeCreditCard` method with `@JobWorker`:
+
+```java
+  @JobWorker(type = "charge-credit-card")
+```
+
+5. Specify a `@Variable(name = "totalWithTax") Double totalWithTax` argument into the `chargeCreditCard` method:
+
+```java
+  public Map<String, Double> chargeCreditCard(@Variable(name = "totalWithTax") Double totalWithTax) {
+```
+
+6. Lastly, implement the `chargeCreditCard` method:
+
+```java
+   LOG.info("charging credit card: {}", totalWithTax);
+    return Map.of("amountCharged", totalWithTax);
```

Comment:
```suggestion
    return Map.of("amountCharged", totalWithTax);
  }
}
```

I'm not sure if this is how you want to handle it, but we're telling them to write code that won't compile. We've included the opening braces in the earlier snippets; we need to tell them to write the closing braces. Many people would see the compiler error and add them, but some will stop immediately at seeing a compiler error and open an issue or ask for help.

The alternative approach, and I think my personal preference, would be to give them the entire class in one snippet, and _then_ explain what's happening in snippet. This allows them to copy 1-2 pieces of code instead of 4-5. (1 if we include the imports in the code snippet; 2 if we separate that out.)

Reply:
I tried to clean this up in my latest commit -- is this closer to what you'd like to see?

Reply:
Yes, I think this format is less likely to result in compilation errors! Thank you!